src/middleware.js

```diff
@@ -158,6 +162,12 @@ const getLinks = payload => {
   };
 };
 
+function saveMeta(action, dispatch) {
+  const { schema, tag } = action.meta;
+  const meta = getMeta(action.payload);
+  dispatch(makeIndexAction(action, REFERENCE_STATUS, { meta }, schema, tag));
```

Comment:
check if meta exists before dispatching empty object. Same for `saveLinks` in ln 174. Because of `saveLinks` change check what will happen when reducers dont receive empty `links`. Probably `getCollectionLink` should return same value.